### ISSUE TITLE
Make some coding-standard improvements to twisted.trial.test.test_util.TestMktemp

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#5857]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)| @exarkun reported|
|-|-|
|Trac ID|trac#5857|
|Type|enhancement|
|Created|2012-08-13 12:44:16Z|
|Branch|https://github.com/twisted/twisted/tree/testmktemp-cleanups-5857|

This `TestCase` has tests without docstrings and using some of the assertion methods we're trying to deprecate.

Split off from [#5853](https://github.com/twisted/twisted/issues/5853).

<details><summary>Searchable metadata</summary>

```
trac-id__5857 5857
type__enhancement enhancement
reporter__exarkun exarkun
priority__normal normal
milestone__ 
branch__branches_testmktemp_cleanups_5857 branches/testmktemp-cleanups-5857
branch_author__exarkun exarkun
status__closed closed
resolution__fixed fixed
component__trial trial
keywords__ 
time__1344861856000000 1344861856000000
changetime__1344869314000000 1344869314000000
version__None None
owner__exarkun exarkun
cc__jml
```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_2"></a>|@exarkun commented|
|-|-|

(In [35163]) Branching to 'testmktemp-cleanups-5857'

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_3"></a>|@exarkun commented|
|-|-|

(In [35164]) Add docstrings and whitespace, switch to preferred assertion methods

refs [#5857](https://github.com/twisted/twisted/issues/5857)

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_4"></a>|@exarkun removed owner|
|-|-|

Done, simple, [build results](http://buildbot.twistedmatrix.com/boxes-supported?branch=/branches/testmktemp-cleanups-5857).

**Comment by twisted-trac** [comment]
|[<img alt="Julian's avatar" src="https://avatars.githubusercontent.com/u/329822?s=50" width="50" height="50">](https://github.com/Julian)<a name="note_5"></a>|@Julian set owner to @exarkun|
|-|-|

Looks good.

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_6"></a>|@exarkun set status to `closed`|
|-|-|

(In [35174]) Merge testmktemp-cleanups-5857

Author: exarkun
Reviewer: Julian
Fixes: [#5857](https://github.com/twisted/twisted/issues/5857)

Adjust the unit tests for `twisted.trial.unittest.TestCase.mktemp` to use the now
preferred spelling of various assertion methods and to have docstrings for each
test method.
